example/src/main/kotlin/com.expedia.graphql.sample/dataFetchers/SpringDataFetcherFactory.kt

```diff
@@ -22,5 +24,12 @@ class SpringDataFetcherFactory: DataFetcherFactory<Any>, BeanFactoryAware {
         val targetedTypeName = environment?.fieldDefinition?.type?.deepName?.removeSuffix("!")?.removeSuffix("Input")
         return beanFactory.getBean("${targetedTypeName}DataFetcher") as DataFetcher<Any>
     }
+}
 
-}
\ No newline at end of file
+// keeping it internal
+internal val GraphQLType.deepName: String
```

Comment:
Do we need this to be exposed for the SDK in enterprise?

Reply:
no idea, think it was just used as a simple example